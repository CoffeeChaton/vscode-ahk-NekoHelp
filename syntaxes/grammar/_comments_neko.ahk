;@ahk-neko-ignore 1 line
;@ahk-neko-ignore 2 line
;@ahk-neko-ignore 999 line

;use 0 to open diag
;@ahk-neko-ignore 0 line
;@ahk-neko-ignore-fn 1 line
;@ahk-neko-ignore-fn 2 line
;@ahk-neko-ignore-fn 999 line

;use 0 to open diag
;@ahk-neko-ignore-fn 0 line
;@ahk-neko-format-ignore-start
;@ahk-neko-format-ignore-end
;@ahk-neko-format-ignore-start
;@ahk-neko-format-ignore-end
;@ahk-neko-format-inline-spacing-ignore-start
;@ahk-neko-format-inline-spacing-ignore-end
;@ahk-neko-format-inline-spacing-ignore-start
;@ahk-neko-format-inline-spacing-ignore-end


;@ahk-neko-color-picker-ignore 1 line

/*@ahk-neko-format-ignore-block
* something
*/