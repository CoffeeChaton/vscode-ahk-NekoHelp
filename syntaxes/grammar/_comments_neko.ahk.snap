>;@ahk-neko-ignore 1 line
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                 ^ source.ahk comment.line.semicolon.ahk
#                  ^ source.ahk comment.line.semicolon.ahk constant.numeric.ahk
#                   ^^^^^ source.ahk comment.line.semicolon.ahk
>;@ahk-neko-ignore 2 line
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                 ^ source.ahk comment.line.semicolon.ahk
#                  ^ source.ahk comment.line.semicolon.ahk constant.numeric.ahk
#                   ^^^^^ source.ahk comment.line.semicolon.ahk
>;@ahk-neko-ignore 999 line
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                 ^ source.ahk comment.line.semicolon.ahk
#                  ^^^ source.ahk comment.line.semicolon.ahk constant.numeric.ahk
#                     ^^^^^ source.ahk comment.line.semicolon.ahk
>
>;use 0 to open diag
#^ source.ahk comment.line.semicolon.ahk punctuation.definition.comment.ahk
# ^^^^^^^^^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
>;@ahk-neko-ignore 0 line
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                 ^ source.ahk comment.line.semicolon.ahk
#                  ^ source.ahk comment.line.semicolon.ahk constant.numeric.ahk
#                   ^^^^^ source.ahk comment.line.semicolon.ahk
>;@ahk-neko-ignore-fn 1 line
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                    ^ source.ahk comment.line.semicolon.ahk
#                     ^ source.ahk comment.line.semicolon.ahk constant.numeric.ahk
#                      ^^^^^ source.ahk comment.line.semicolon.ahk
>;@ahk-neko-ignore-fn 2 line
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                    ^ source.ahk comment.line.semicolon.ahk
#                     ^ source.ahk comment.line.semicolon.ahk constant.numeric.ahk
#                      ^^^^^ source.ahk comment.line.semicolon.ahk
>;@ahk-neko-ignore-fn 999 line
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                    ^ source.ahk comment.line.semicolon.ahk
#                     ^^^ source.ahk comment.line.semicolon.ahk constant.numeric.ahk
#                        ^^^^^ source.ahk comment.line.semicolon.ahk
>
>;use 0 to open diag
#^ source.ahk comment.line.semicolon.ahk punctuation.definition.comment.ahk
# ^^^^^^^^^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
>;@ahk-neko-ignore-fn 0 line
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                    ^ source.ahk comment.line.semicolon.ahk
#                     ^ source.ahk comment.line.semicolon.ahk constant.numeric.ahk
#                      ^^^^^ source.ahk comment.line.semicolon.ahk
>;@ahk-neko-format-ignore-start
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                 ^^^^^^^^ source.ahk comment.line.semicolon.ahk
#                         ^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
>;@ahk-neko-format-ignore-end
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                 ^^^^^^^^ source.ahk comment.line.semicolon.ahk
#                         ^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
>;@ahk-neko-format-ignore-start
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                 ^^^^^^^^ source.ahk comment.line.semicolon.ahk
#                         ^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
>;@ahk-neko-format-ignore-end
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                 ^^^^^^^^ source.ahk comment.line.semicolon.ahk
#                         ^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
>;@ahk-neko-format-inline-spacing-ignore-start
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                 ^^^^^^^^^^^^^^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#                                        ^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
>;@ahk-neko-format-inline-spacing-ignore-end
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                 ^^^^^^^^^^^^^^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#                                        ^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
>;@ahk-neko-format-inline-spacing-ignore-start
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                 ^^^^^^^^^^^^^^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#                                        ^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
>;@ahk-neko-format-inline-spacing-ignore-end
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                 ^^^^^^^^^^^^^^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#                                        ^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk